enhancements/kubelet/virtualization-higher-workload-density.md

```diff
@@ -0,0 +1,551 @@
+---
+title: virtualization-higher-workload-density
+authors:
+  - "@enp0s3"
+  - "@iholder101"
+  - "@fabiand"
+reviewers:
+  - "@rphilips"
+  - "@sjenning"
+approvers:
+  - "@mrunalp"
+api-approvers:
+  - "@mrunalp"
+creation-date: "2024-05-05"
+last-updated: "2024-06-12"
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - "https://issues.redhat.com/browse/CNV-28178"
+see-also:
+  - "https://github.com/kubernetes/enhancements/issues/2400"
+status: implementable
+---
+
+# Virtualization higher workload density
+
+## Summary
+
+Fit more workloads onto a given node - achieve a higher workload
+density - by overcommitting it's memory resources. Due to timeline
+needs a two-phased approach is considered.
+
+## Motivation
```

Comment:
Maybe we should mention memory types that are often underused in the guest, such as cache memory, unused application memory, and idle memory.